src/main/kotlin/no/nav/permitteringsskjemaapi/kafka/PermitteringsskjemaProdusent.kt

```diff
@@ -41,17 +46,24 @@ class PermitteringsskjemaProdusent(
             yrkeskategorier = permitteringsskjema.yrkeskategorier,
         )
 
+        val jsonEvent = mapper.writeValueAsString(rapport)
+
+        if (validateAgainstSchema(jsonEvent).isNotEmpty()) {
+            /* consider logging errors */
+            log.error("invalid json schema")
```

Comment:
burde vi ikke feile hardt her så vi ikke får ting på topic som ikke er i henhold til skjema? 
Feilmeldingen sier forhåpentligvis ikke noe om faktisk verdi, men bare hvilken regel den bryter, så er nok fint å logge den ja.

Reply:
hmm, jo, kanskje det. Tanker var og la den kjøre litt, for å se om skjemaet matcher det som blir sendt. Så kunne man endret til å feile senere.

Hvis vi endrer den til å kaster nå, så vil noen fra batchen bli sendt, men transaksjonen rulles tilbake, så det blir litt dobbelt-sending osv. Det gjør sånn sett ikke noe, da.

Reply:
Lagde sak: https://jira.adeo.no/browse/TAG-2184